cmd/kubelet/app/server.go

```diff
@@ -653,7 +653,7 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies) (err error) {
 				ExperimentalQOSReserved:               *experimentalQOSReserved,
 				ExperimentalCPUManagerPolicy:          s.CPUManagerPolicy,
 				ExperimentalCPUManagerReconcilePeriod: s.CPUManagerReconcilePeriod.Duration,
-				ExperimentalPodPidsLimit:              *s.PodPidsLimit,
+				ExperimentalPodPidsLimit:              s.PodPidsLimit,
```

Comment:
This has a default, right?

Reply:
Yes, default is -1: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1alpha1/defaults.go#L132